core/jobs.py

```diff
@@ -1216,13 +1633,25 @@ class JobCleanupManager(BaseMapReduceJobManager):
 
     @classmethod
     def entity_classes_to_map_over(cls):
+        """The entity types this job will handle."""
         return [
             mapreduce_model.MapreduceState,
             mapreduce_model.ShardState
         ]
 
     @staticmethod
     def map(item):
+        """Implements the map function which will clean up jobs that have not
+        finished.
+
+        Args:
+            item: mapreduce_model.MapreduceState or mapreduce_model.ShardState.
+                A shard or job which may still be running.
+        Yields:
+            tuple(str, int). Describes the action taken for the item, and the
+                number (items are countable) of items for which this action was
```

Comment:
Sorry, I meant just "number", not "number (items are countable)". The part in parens was a justification for changing "amount" to "number".